adapters/adform/info.yaml

```diff
@@ -0,0 +1,2 @@
+maintainer:
+  email: "scope.sspp@adform.com"
```

Comment:
Looks like this was added accidentally. This file got moved to `static/bidder-info/adform.yaml`... so you can delete it here.

Reply:
ahh nevermind... apparently I can just delete this through the github UI :)